packages/suite/src/views/wallet/nfts/NftsTable/NftsRow.tsx

```diff
@@ -0,0 +1,268 @@
+import { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Network } from '@suite-common/wallet-config';
+import {
+    DefinitionType,
+    tokenDefinitionsActions,
+    TokenManagementAction,
+    EnhancedTokenInfo,
+} from '@suite-common/token-definitions';
+import { Badge, Button, Dropdown, Icon, Row, Table, Text, useElevation } from '@trezor/components';
+import { borders, Elevation, mapElevationToBackground, spacings } from '@trezor/theme';
+import { SelectedAccountStatus } from '@suite-common/wallet-types';
+import { getNftExplorerUrl, getNftContractExplorerUrl } from '@suite-common/wallet-utils';
+
+import { goto } from 'src/actions/suite/routerActions';
+import {
+    HiddenPlaceholder,
+    RedactNumericalValue,
+    Translation,
+    TrezorLink,
+} from 'src/components/suite';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { SUITE } from 'src/actions/suite/constants';
+import { selectIsCopyAddressModalShown } from 'src/reducers/suite/suiteReducer';
+import { copyAddressToClipboard, showCopyAddressModal } from 'src/actions/suite/copyAddressActions';
+
+import { BlurUrls } from '../../tokens/common/BlurUrls';
+import { DropdownRow } from '../../tokens/DropdownRow';
+
+type NftsRowProps = {
+    nft: EnhancedTokenInfo;
+    network: Network;
+    selectedAccount: SelectedAccountStatus;
+    isShown?: boolean;
+    isEmptyCollection?: boolean;
+    setIsEmptyCollectionsOpen?: (open: boolean) => void;
+    isEmptyCollectionsOpen?: boolean;
+};
+
+const PictureIconWrapper = styled.div<{ $elevation: Elevation }>`
```

Comment:
Try using IconCircle with hasBorder=false, paddingType=small (or medium, perhaps), size={32}, variant=tertiary instead of custom styles.

Reply:
Thank you, I was wondering if there's a component like that. It's perfect